Update readme (#28)
gnosis/gp-ui: README.md

[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1/badge.svg?branch=master)](https://coveralls.io/github/gnosis/gp-v1?branch=master)

Develop:
[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1/badge.svg?branch=develop)](https://coveralls.io/github/gnosis/gp-v1?branch=develop)

> # Gnosis Protocol UI

Reviewer: I'd say keep `v1` in the header

Author: why? this is not V1 anymore, the default app now is the explorer

Reviewer: Oh, that's what you mean.
I thought it was still indicating the former `dex-react` app.

> `Gnosis Protocol UI` contains some reusable components for [Gnosis Protocol](https://docs.gnosis.io/protocol) (`🚨 WARNING: the doc might refer still to v1 of the protocol`).

Reviewer 2: ```suggestion
`Gnosis Protocol UI` contains some reusable components for [Gnosis Protocol](https://docs.gnosis.io/protocol) (`🚨 WARNING: the doc may still refer to v1 of the protocol`).
```

Gnosis Protocol is a fully permissionless DEX that enables ring trades to maximize liquidity.

`Gnosis Protocol UI` contains:

> - **Explorer**: Gnosis Protocol explorer. Allows to explore the protocol orders, trades. For now is a WIP 👷‍♀️.

Reviewer 2: ```suggestion
- **Explorer**: Gnosis Protocol explorer. Allows you to explore the protocol orders and trades. For now Explorer is a WIP 👷‍♀️.
```

> - **Trade**: Classical trading interface. For now is just a WIP 👷‍♀️. 

Reviewer 2: ```suggestion
- **Trade**: Classical trading interface. For now it is just a WIP 👷‍♀️. 
```

> - **Legacy GP v1 UI**: Trading interface for Gnosis Protocol v1. It's the only app that is not generated automatically, but it can be run or build by following the instructions in this readme.

Reviewer 2: ```suggestion
- **Legacy GP v1 UI**: Trading interface for Gnosis Protocol v1. It's the only app that is not generated automatically, but it can be run/built by following the instructions in this readme.
```

> - **Story Book**: Showcase with some of the components used in these apps.

Reviewer 2: ```suggestion
- **Story Book**: Showcases some of the components used in these apps.
```
